src/shared/ui/Typography/index.ts

```diff
@@ -0,0 +1,15 @@
+export {
+  Title1,
+  Title2,
+  Title3,
+  Caption1,
+  Caption2,
+  Body1,
+  Body2,
+  Body3,
+  Subtitle1,
+  Subtitle2,
+  Subtitle3,
+  ButtonTitle1,
+  ButtonTitle2,
+} from './Typhgraphy';
```

Comment:
```suggestion
} from './Typography';
```
새로 만드신 Typography 요 파일로 변경되는게 맞을까요?!

Reply:
e9580f94688dc7d2249c4a02f21cf281be1ff30f 수정하였습니다!!